README.rst

```diff
@@ -154,24 +154,26 @@ Developers". Feel free to cite our work in your research using the following Bib
 
 .. code-block::
 
-    @software{pygmt_2021_4522136,
+    @software{pygmt_2021_4592991,
       author       = {Uieda, Leonardo and
                       Tian, Dongdong and
                       Leong, Wei Ji and
                       Toney, Liam and
                       Schlitzer, William and
                       Grund, Michael and
+                      Yao, Jiayuan and
```

Comment:
```suggestion
                      Schlitzer, William and
                      Yao, Jiayuan and
                      Grund, Michael and
                      Toney, Liam and
```

Reply:
I had based this off number of commits yesterday; I'll put  
`` Toney, Liam and
Schlitzer, William and
Yao, Jiayuan and
Grund, Michael and
``

In the v0.3.0 release we put @liamtoney up higher because of his ROSES video, which may not count towards his commits but has significantly enhanced the project. Ping @GenericMappingTools/python for input, but I think he should remain where he is in the order because he is spearheading the SciPy session.

Reply:
> Following @weiji14 , I checked [Citing PyGMT](https://pygmt-git-v031-release-gmt.vercel.app/index.html#citing-pygmt) and authorship on Zenodo archives of releases at [AUTHORSHIP.md](https://github.com/GenericMappingTools/pygmt/blob/master/AUTHORSHIP.md):
> 
> > The order of authors will be defined by the number of commits to the repository (git shortlog -sne). The order can also be changed on a case-by-case basis.

Yes, on a first order, we sort things by the number of commits. But pay attention to the next sentence "The order can also be changed on a case-by-case basis". The idea is that contributions are not just about commits but also the other things people do. For example, Liam put a lot of work into the ROSES tutorial (the video on the front page) and Meghan has done lots of things behind the scenes such as funding applications and setting up accounts.

I don't want to get too fired up about name ordering, but it's really up to you all to negotiate what you think is best. Just stating that there are other factors to consider :)

Reply:
Changes pushed in 8fef314

Reply:
Good. That's reasonable.

Reply:
>  Actually, I begin to know PyGMT from his ROSES tutorial. 

Me too! Thanks Liam!

Reply:
Everythin fine for me 😉.